content/en/docs/tasks/debug/debug-cluster/troubleshoot-kubectl-connectivity.md

```diff
@@ -0,0 +1,110 @@
+---
+reviewers:
+# Should be added by an experienced contributor
+title: "Troubleshooting kubectl Connectivity"
+content_type: task
+weight: 10
+---
+
+<!-- overview -->
+
+This documentation is about investigating and diagnosing connectivity issues between
+{{<glossary_tooltip text="kubectl" term_id="kubectl">}} and your Kubernetes cluster.
+If you are unable to connect to your cluster using kubectl, this document describes
+several common scenarios and potential solutions to identify and address the likely
+cause.
+
+Before proceeding with the troubleshooting guide, please ensure that you have already
```

Comment:
Please consider following the content style guide for task pages.
We often have a 'prerequisite' section dedicated to things like this.